Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,378 @@
+        label.layer.cornerRadius = 11
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 13)
+        label.textAlignment = .center
+        return label
+    }()
+    private var missionBackgroundView: UIView = {
+        var view = UIView()
+        view.backgroundColor = .darkGrey004
+        view.layer.borderWidth = 1
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private let missionTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "오늘의 개별 미션"
+        label.textColor = .darkGrey002
+        label.font = .font(.regular, ofSize: 14)
+        return label
+    }()
+    private lazy var missionContentsLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 18)
+        return label
+    }()
+    private let informationTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "진행 정보"
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private let manitteeBackView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .darkGrey002
+        view.makeBorderLayer(color: .white)
+        return view
+    }()
+    private let manitteeImageView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .characterYellow
+        view.layer.cornerRadius = 49.5
+        return view
+    }()
+    private var manitteeIconView: UIImageView = {
+        var imageView = UIImageView()
+        imageView.image = ImageLiterals.icManiTti
+        return imageView
+    }()
+    private lazy var manitteeLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 15)
+        return label
+    }()
+    private let listBackView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .darkGrey002
+        view.makeBorderLayer(color: .white)
+        return view
+    }()
+    private let listImageView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .characterPink
+        view.layer.cornerRadius = 49.5
+        return view
+    }()
+    private let listIconView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = ImageLiterals.icList
+        return imageView
+    }()
+    private let listLabel: UILabel = {
+        let label = UILabel()
+        label.text = "함께하는 친구들"
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 15)
+        return label
+    }()
+    private lazy var letterBoxButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle("쪽지함", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.titleLabel?.font = .font(.regular, ofSize: 15)
+        button.backgroundColor = .darkGrey002
+        button.makeBorderLayer(color: .white)
+        return button
+    }()
```

Comment:
p3;

property 생성 단계에서 UIAction을 넣어주세요 !

```suggestion
    private lazy var letterBoxButton: UIButton = {
        let button = UIButton(type: .system)
        let action = UIAction { _ in
            print("쪽지함!!")
        }
        button.setTitle("쪽지함", for: .normal)
        button.setTitleColor(.white, for: .normal)
        button.titleLabel?.font = .font(.regular, ofSize: 15)
        button.backgroundColor = .darkGrey002
        button.makeBorderLayer(color: .white)
        button.addAction(action, for: .touchUpInside)
        return button
    }()
```

Reply:
> p3;
> 
> property 생성 단계에서 UIAction을 넣어주세요 !

2a0adc4 감사합니다~